battleship-server/src/main/java/com/epam/solid/nie/config/language/CommunicateProviderImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.solid.nie.config.language;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+public class CommunicateProviderImpl implements CommunicateProvider {
```

Comment:
I don't like names with "Impl" part in it. Maybe better name would be FileCommunicateProvider

Reply:
we will do this in the future commit